src/controllers/commands/CommandController.cpp

```diff
@@ -1195,6 +1196,87 @@ void CommandController::initialize(Settings &, Paths &paths)
             crossPlatformCopy(words.mid(1).join(" "));
             return "";
         });
+
+    this->registerCommand("/color", [](const QStringList &words, auto channel) {
+        auto user = getApp()->accounts->twitch.getCurrent();
+
+        // Avoid Helix calls without Client ID and/or OAuth Token
+        if (user->isAnon())
+        {
+            channel->addMessage(makeSystemMessage(
+                "You must be logged in to use the /color command"));
+            return "";
+        }
+
+        // Colors retreived from https://dev.twitch.tv/docs/api/reference#update-user-chat-color 2022-09-11
+        const QStringList validColors{
+            "blue",      "blue_violet",  "cadet_blue",   "chocolate",
+            "coral",     "dodger_blue",  "firebrick",    "golden_rod",
+            "green",     "hot_pink",     "orange_red",   "red",
+            "sea_green", "spring_green", "yellow_green",
+        };
+
+        auto colorString = words.value(1);
+
+        if (colorString.isEmpty())
+        {
+            channel->addMessage(makeSystemMessage(
+                QString("Usage: /color <color> - Color must be one of Twitch's "
+                        "supported colors (%1) or a hex code (#000000) if you "
+                        "have Turbo or Prime.")
```

Comment:
```suggestion
                        "have Turbo.")
```

Despite what the twitch docs say, non-turbo prime users aren't able to set a custom hex code

Reply:
I tested this incorrectly; the command *does* work for prime & this is now an advertised feature of prime https://www.amazongames.com/en-us/support/prime-gaming/articles/what-is-prime-gaming

https://github.com/Chatterino/chatterino2/pull/3988/commits/fd8ae3f1cf1cd8039afe7a59755b32e5e6e875d2 can be reverted